pkgs/tools/misc/whatsapp-chat-exporter/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+
+, withEncryptedAndroid ? false
+, withCrypt12 ? false
+, withCrypt14 ? false
+, withCrypt15 ? false
+}:
+
+let version = "0.9.1";
+in python3Packages.buildPythonApplication {
+  pname = "whatsapp-chat-exporter";
+  inherit version;
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "KnugiHK";
+    repo = "Whatsapp-Chat-Exporter";
+    rev = "refs/tags/${version}";
+    hash = "sha256-DvCYMfR9GgdP9rVpcoIR5nG9b4ToOBMG1a9OTvjfIiU=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    bleach
+    jinja2
+  ] ++ lib.optionals (withEncryptedAndroid || withCrypt15) [
+    pycryptodome
+    javaobj-py3
+  ] ++ lib.optionals (withCrypt12 || withCrypt14) [
+    pycryptodome
```

```suggestion
    javaobj-py3
    pycryptodome
```
we can't just do this and remove all options from this package?

---

sure. My thought was to mirror the installation options of the python package in [its setup.py](https://github.com/KnugiHK/Whatsapp-Chat-Exporter/blob/0.9.1/setup.py#L52-L57)

---

We don't need to that. It only adds confusion here.